receiver/kubeletstatsreceiver/README.md

```diff
@@ -285,3 +285,18 @@ rules:
     resources: ["nodes/proxy"]
     verbs: ["get"]
 ```
+
+### Warning about metrics' deprecation
+
+The following metrics will be removed in `v0.111.0` version:
+- `k8s.node.cpu.utilization` (replaced by `k8s.node.cpu.usage`)
+- `k8s.pod.cpu.utilization` (replaced by `k8s.pod.cpu.usage`)
+- `container.cpu.utilization` (replaced by `container.cpu.usage`)
+
+Users can use the following metrics instead:
+- `k8s.node.usage`
+- `k8s.pod.usage`
+- `container.usage`
```

Comment:
Out of ignorance, what's the relevance of this section? The comment above shares which metrics are being removed, and what metric they're being replaced by. This appears to be somewhat duplicating the "replaced by" message, but with different metrics. (I very well may be missing something, just want to check.)

Reply:
Good catch! Actually there was a `*.cpu.*` part missing by mistake there. Thank's for spotting this!